frontend/src/containers/Ledger.js

```diff
@@ -157,9 +159,19 @@ export class Ledger extends Component {
       fetchUsers,
       fetchPendingExpenses,
       fetchTransactions,
-      loadData
+      loadData,
+      params
     } = this.props;
 
+    switch (params.action) {
+      case 'approve':
+        this.approveExp(params.expenseid);
+        break;
+      case 'reject':
+        this.rejectExp(params.expenseid);
+        break;
+    }
+
```

Comment:
Ok. Fine for now. Something like `notify('success', 'Expense approved')` in the approve promise chain would be useful.